speed-dreams/source-2.2.3/src/modules/racing/standardgame/racesituation.h

```diff
@@ -82,7 +82,7 @@ class ReSituation
     //! The mutex for thread-safe access.
     struct SDL_mutex* _pMutex;
 
-    // ReSituationUpdater is a very good friend 8-D.
+    // ReSituationUpdater is a very good friend.
```

Comment:
Any reason you randomly edit this?

Reply:
It is a pictorgram of a male bodypart and is not very professional